environments/production/common/route53.tf

```diff
@@ -1,18 +1,22 @@
+resource "aws_route53_zone" "this" {
```

Comment:
We don't need this I don't think

Reply:
Maybe add a comment just to say this was imported via CLI

Reply:
You're right! _This_ one should be a data source.